packages/app/src/components/nominationBanner/style.css

```diff
@@ -76,3 +76,116 @@
 .hippa-column {
   white-space: nowrap;
 }
+
+
+/* modal style */
+
+.modal-background{
+  opacity: 1;
+  pointer-events: none;
+  transition: opacity 0.4s case-in-out ;
+  position: absolute; 
+  
+  z-index: 1000;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.75);
+}
+
+
+/* .modalShowing-true {
+  opacity: 1;
+  pointer-events: visible;
+} */
```

Comment:
If we're not using this anymore, we should probably just remove it.

Reply:
Done